runtime/backtrace.c

```diff
@@ -374,3 +381,22 @@ CAMLprim value caml_get_current_callstack(value max_frames_value)
   caml_stat_free(callstack);
   CAMLreturn(res);
 }
+
+void caml_print_current_callstack(int max_frames_value)
```

Comment:
Nitpicking, but I would call `max_frames_value` something like `num_frames` since it is not actually a "value".